new-feature.txt

```diff
@@ -0,0 +1 @@
+This file was created in feature1 branch
```

Comment:
its really good that you have decided to add the code

Reply:
Thanks for comment githubber